src/scripts/runscripts.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+COUNTER=0 # The count of tests run
+FAIL=0 # The count of failed tests
+SUCCESS=0 # The count of successful tests
+for i in $(ls $1); do
+	COUNTER=$(($COUNTER+1))
+	$2 $1/$i 2>/dev/null
+	ERROR_CODE=$?
+	if [ $ERROR_CODE == 0 ] ; then
+		tput setaf 2
+		SUCCESS=$(($SUCCESS+1))
+	else
+		if [ $ERROR_CODE == 120 ] ; then
+			ERROR_CODE=$ERROR_CODE" assertion failed"
+		fi
+		tput setaf 1
+		FAIL=$(($FAIL+1))
+	fi
+	echo $i "  Exit code: " $ERROR_CODE
```

Comment:
Also a '-q' options would be great. If this option would be enabled only the final stats would be printed.